app/models/post_custom_prompt.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class PostCustomPrompt < ActiveRecord::Base
+  belongs_to :post
+end
+
+class ::Post
+  has_one :post_custom_prompt, dependent: :destroy
```

Comment:
Core extensions should live inside the `EntryPoint`

Reply:
tricky cause I don't think we have a "common" entry point, maybe in `plugin.rb` ?